Composer/ScriptHandler.php

```diff
@@ -144,6 +144,22 @@ public static function clearCache(Event $event)
     }
 
     /**
+     * Warm up the Symfony cache.
+     *
+     * @param Event $event
+     */
+    public static function warmupCache(Event $event)
+    {
+        $options = static::getOptions($event);
+        $consoleDir = static::getConsoleDir($event, 'Warm up the cache');
```

Comment:
(3/3) Insert a blank line between this and the `if`, to keep consistent with `clearCache` and `installAssets`.